CONTRIBUTING.rst

```diff
@@ -140,13 +140,12 @@ in the dask-image repository.
 There is a separate environment file for each supported Python version.
 
 We will use conda to
-`create an environment from a file
-<https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#creating-an-environment-from-an-environment-yml-file>`_
+`create an environment from a file`_
 (``conda env create -f name-of-environment-file.yml``).
 
 .. note::
-    If you do not have Anaconda/miniconda installed, please follow
-    `these instructions <https://docs.conda.io/projects/conda/en/latest/user-guide/install/>`_.
+    If you don't have `conda`_ installed, we recommend downloading and installing it
+    with the conda-forge distribution `Miniforge here`_.
```

Comment:
Spotted the same issue in `CONTRIBUTING.rst`. The text looked the same as `docs/installation.rst`. So copied your text over here